src/content/components/BugListView/BugListView.js

```diff
@@ -79,6 +101,9 @@ export class BugListView extends React.PureComponent {
           tags={true}
           bugs={this.state.bugs}
           columns={this.props.columns}
+          fetchBugs={() => {
+            this.fetchBugs();
+          }}
```

Comment:
this is going to recreate the function on every render. just use `fetchBugs={this.fetchBugs}`, and then in the constructor add `this.fetchBugs = this.fetchBugs.bind(this)`

in functional components, you can do `const fetchBugs = React.useCallback(() => {})` and pass that instead